doc/release-notes/0.11/0.11.md

```diff
@@ -0,0 +1,164 @@
+
+Builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.11.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.11/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3118">#3118</a></td>
+<td valign="top">OpenSSL V1.1.1 support</td>
+<td valign="top">OpenJDK8 (All platforms)</td>
+<td valign="top">New command line options are available to enable OpenSSL support for improved cryptographic performance.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1646">#1646</a></td>
+<td valign="top">The shared classes cache is now enabled by default for bootstrap classes</td>
+<td valign="top">OpenJDK11 (All platforms)</td>
+<td valign="top">In earlier releases, the shared classes cache was enabled by specifying `-Xshareclasses` on the command line. This option shares both the bootstrap and application classes that are loaded by the default system class loader. Bootstrap classes are now shared by default.</td>
+</tr>
+
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/2862">#2862</a></td>
+<td valign="top">The default shared cache and cache snapshot directory is changed to the user's home directory</td>
+<td valign="top">OpenJDK11 (AIX, Linux)</td>
+<td valign="top">This change is for non-Windows platforms and applies only if `-Xshareclasses:groupAccess` is not used. If `groupAccess` is used, the directory that gets set remains as `/tmp/javasharedresources/`.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/2862">#2862</a></td>
+<td valign="top">Two new `-Xshareclasses` suboptions are introduced </td>
+<td valign="top">OpenJDK8 and later (All platforms)</td>
+<td valign="top">`-Xshareclasses:bootClassesOnly` prevents non-bootstrap class loaders caching classes in the shared classes cache. `-Xshareclasses:fatal` prevents the shared classes cache being started if an error occurs. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3184">#3184</a></td>
+<td valign="top">VM container-awareness is now enabled by default</td>
+<td valign="top">OpenJDK8 and later (All platforms)</td>
```

Comment:
@ashu-mehra should this be Linux instead of All Platforms?

Reply:
Yes, it should be Linux.